src/lib/components/Popup.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	import { ChevronRightIcon } from 'svelte-feather-icons';
+
+	export let id: string;
+	export let label: string;
+</script>
+
+<div class="popup">
+	<input {id} type="checkbox" hidden />
+	<label for={id}>
+		{label}
+		<ChevronRightIcon class="inline" />
```

Comment:
Perhaps investigate a better placement, and maybe not embedding the SVG into the HTML.